local-cli/generator/templates.js

```diff
@@ -13,42 +13,6 @@ const execSync = require('child_process').execSync;
 const fs = require('fs');
 const path = require('path');
 
-/**
- * Templates released as part of react-native in local-cli/templates.
- */
-const builtInTemplates = {
```

Comment:
Line 44 in the edited file, or 80 in the original, contains a reference to this `builtInTemplates` const. This was picked up by lint during import.